Content.Shared/GameObjects/EntitySystems/SharedPullingSystem.cs

```diff
@@ -198,5 +218,25 @@ public bool IsPulling(IEntity puller)
         {
             return _pullers.ContainsKey(puller);
         }
+
+        private void UpdatePulledRotation(IEntity puller, IEntity pulled)
+        {
+            if (!pulled.TryGetComponent(out SharedRotatableComponent? rotatable))
+                return;
+
+            if (!rotatable.RotateWhilePulling)
+                return;
+
+            var dir = puller.Transform.WorldPosition - pulled.Transform.WorldPosition;
+            if (dir.LengthSquared > ThresholdRotDistance * ThresholdRotDistance)
+            {
+                var oldAngle = pulled.Transform.LocalRotation;
+                var newAngle = Angle.FromWorldVec(dir);
+
+                var diff = newAngle - oldAngle;
+                if (Math.Abs(diff.Degrees) > ThresholdRotAngle)
+                    pulled.Transform.LocalRotation = newAngle;
```

Comment:
Should this be comparing local angles with world angles? Won't it look weird with rotated shuttles?